### PR TITLE
Update css, add dark mode

### PR DESCRIPTION
**Pull request by nspenner**
Switched some fonts, added dark mode support. Added spacing between episodes and removed some horizontal lines.

style_updates
style_updates_dark

### CODE REVIEW

File: src/app.html [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
-		<meta name="viewport" content="width=device-width" />
-		%sveltekit.head%
-	</head>
-	<body data-sveltekit-preload-data="hover">
-		<div style="display: contents">%sveltekit.body%</div>
-	</body>
-</html>
+
+<head>
+	<meta charset="utf-8" />``
```
Comment (nminchow) [inline src/app.html:5]: rogue backticks

File: src/app.html [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
+		rel="stylesheet">
+	<meta name="viewport" content="width=device-width" />
+	%sveltekit.head%
+</head>`
+
+<body data-sveltekit-preload-data="hover">
+	<div style="display: contents">%sveltekit.body%</div>
+</body>
+
+</html>
```
Comment (nminchow) [inline src/app.html]: probably should \n while we are here.

**Review by nminchow — COMMENTED** [review]
👨‍🎨

File: src/app.html (hunk 2 above)
Comment (nspenner) [reply src/app.html]: weird that the default html formatter in vscode doesn't do it, but fixed

Comment (nminchow) [reply src/app.html]: yeah, there are other files missing them as well

**Pull request merged**